<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>700</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>pip Graphical Manager</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>781</width>
      <height>681</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="2" column="0" colspan="3">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPushButton" name="installButton">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Install Package</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="uninstallButton">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Uninstall Package</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="updateButton">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Update Package</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="updatePipButton">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="text">
          <string>Update pip</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="0" column="0">
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLineEdit" name="packageSearchEntry">
           <property name="font">
            <font>
             <pointsize>18</pointsize>
            </font>
           </property>
           <property name="placeholderText">
            <string>Search Packages...</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="searchButton">
           <property name="text">
            <string>Search</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QTableWidget" name="packageTable">
         <property name="verticalScrollBarPolicy">
          <enum>Qt::ScrollBarAlwaysOn</enum>
         </property>
         <property name="editTriggers">
          <set>QAbstractItemView::NoEditTriggers</set>
         </property>
         <attribute name="horizontalHeaderDefaultSectionSize">
          <number>383</number>
         </attribute>
         <column>
          <property name="text">
           <string>Package Name</string>
          </property>
         </column>
        </widget>
       </item>
      </layout>
     </item>
     <item row="1" column="0" colspan="3">
      <widget class="QTextEdit" name="packageInfoTextArea">
       <property name="verticalScrollBarPolicy">
        <enum>Qt::ScrollBarAlwaysOn</enum>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="installedPackagesLabel">
         <property name="font">
          <font>
           <pointsize>18</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Installed Packages</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTableWidget" name="installedPackagesTable">
         <property name="verticalScrollBarPolicy">
          <enum>Qt::ScrollBarAlwaysOn</enum>
         </property>
         <property name="editTriggers">
          <set>QAbstractItemView::NoEditTriggers</set>
         </property>
         <property name="selectionBehavior">
          <enum>QAbstractItemView::SelectRows</enum>
         </property>
         <attribute name="horizontalHeaderDefaultSectionSize">
          <number>191</number>
         </attribute>
         <column>
          <property name="text">
           <string>Package Name</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Version</string>
          </property>
         </column>
        </widget>
       </item>
      </layout>
     </item>
     <item row="0" column="1">
      <widget class="Line" name="line">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
